Governance/introduction.md

```diff
@@ -0,0 +1,21 @@
+# Introduction
+
+This book, '**Climate Dataset Guidelines**', provides guidelines for creating, managing, sharing and publishing climate data. With a focus on Australian climate research it has been designed to enable a common approach across the community. Despite the focus on climate data, many of the principles and tools described within are applicable to many Earth science data applications.
+
+These guidelines are applicable to all levels of data sharing, from peer-to-peer sharing with colleagues to full publication in a public repository. If you expect that even a single other person will use your data for any purpose, including simple checking and comparing, your data should meet FAIR principles and meet a minimum set of formatting and metadata recommendations.
+
+#
+This book covers the follow topics:
+
+### **Creating climate data**
+Creating datasets that are fit for publication can be a daunting process, with many technical aspects to consider. This chapter will outline the ways to ensure that your data meets the broad [FAIR principles](https://ardc.edu.au/resources/working-with-data/fair-data/) of data sharing, along with specific requrirements and recommendations for climate-related data that will aid in the use and reproducibility of your data by the Australian, and international, climate community.
+
+### **Publishing climate data**

```

Comment:
These section headers could be URLs/anchor links pointing to the relevant pages?